source/winAPI/_powerTracking.py

```diff
@@ -124,14 +127,28 @@ def initialize():
 	return
 
 
-def reportCurrentBatteryStatus(onlyReportIfStatusChanged: bool = False) -> None:
+@unique
+class ReportContext(Enum):
 	"""
-	@param onlyReportIfStatusChanged: sometimes multiple events may fire for a power status change.
-	Set this to True to only report if the power status changes.
+	Used to determine the order of information, based on relevance to the user,
+	when announcing power status information
+	"""
+
+	AC_STATUS_CHANGE = auto()
+	"""e.g. a charger is connected/disconnected"""
+	FETCH_STATUS = auto()
+	"""e.g. when a user presses nvda+shift+b to fetch the current battery status"""
+
+
+def reportCurrentBatteryStatus(context: ReportContext) -> None:

```

Comment:
Rather than exposing a variable, I think it would be clearer to have two methods:
- `reportBatteryStateChange`
- `reportCurrentBatteryStatus`

Reply:
I've fixed this by adding wrappers